test/JDBC/expected/BABEL-725-vu-prepare.out

```diff
@@ -0,0 +1,206 @@
+CREATE VIEW BABEL_725_vu_prepare_v1 AS (SELECT ATN2(2, 3));
+GO
+
+-- test with different datatype
+CREATE VIEW BABEL_725_vu_prepare_v2 AS (
```

Comment:
We need to test all data types that can be implicitly casted to `float` (ref: https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/data-types/data-type-conversion-database-engine.md)